src/FrameworkBridge/Symfony/Config/services.php

```diff
@@ -22,6 +23,11 @@
         \dirname(__DIR__, 4) . '/src/Handler/Handler/*'
     );
 
+    $services->set(SymfonyCacheHandler::class)
```

Comment:
why is this needed ? it will be detected and declared automatically I guess

Reply:
Symfony users may encounter autowiring error if they don't have an instance of TagAwareAdapterInterface declared in their services. 
I was assuming it is their concern to declare the handler they want to use, and then let the compiler pass do the rest of the job

Reply:
I see let's keep it like this for now, maybe in the future we'll handle a more advanced auto discovery